.github/workflows/release-dev.yaml

```diff
@@ -95,6 +95,7 @@ jobs:
           
           rm e2e/kots-release-install/release.yaml
           replicated release create --yaml-dir e2e/kots-release-install --promote CI-airgap --version "appver-${SHORT_SHA}"
+          replicated release create --yaml-dir e2e/kots-release-upgrade --promote CI-airgap --version "appver-${SHORT_SHA}-upgrade"
```

Comment:
this workflow is what gets run on merge - it'll need the same set of changes as `.github/workflows/pull-request.yaml`

we should combine them at some point, I think they're nearly identical